flink-libraries/flink-ml/src/main/scala/org/apache/flink/ml/recommendation/ALS.scala

```diff
@@ -273,6 +308,14 @@ object ALS {
     val defaultValue: Option[Int] = Some(10)
   }
 
+  case object ImplicitPrefs extends Parameter[Boolean] {
```

Comment:
I also changed this, but note that this is for the explicit ALS algorithm too. Do you think it's okay to give this recommendation for explicit case too? Really high number of factors (without regularization) might lead to overfitting.